### DISCUSSION TITLE
how to add body background color

### DISCUSSION THREAD

**Discussion by vytautas-pranskunas-** [taiga-family/taiga-ui#9775] [Q&A] [answered]
tui-root has border image. Why it is for? when i add body: background-color it is not visible because of border-image

**Comment by vytautas-pranskunas-** [comment]
btw - why support / response rate has dropped so much? Can anybody answer. Previously you guys were so active - this was the main key point why we have chosen your library...

**Reply by vladimirpotekhin** [reply]
Half the team is on vacation until December, support may be slower during that time

**Comment by vladimirpotekhin** [comment, marked as answer]
Hi, just add border-image: none to the tui-root.
It's made that way so that even if the height of the content is smaller than the screen, tui-root would have a default background and toggling dark theme would paint whole page dark.
